contributing/getting-started-e2e.md

```diff
@@ -153,21 +153,18 @@ Now that you have all of the files copied locally you can actually build the cod
 
 amphtml uses Node.js, the Yarn package manager and the Gulp build system to build amphtml and start up a local server that lets you try out your changes.  Installing these and getting amphtml built is straightforward:
 
-* Install [NodeJS](https://nodejs.org/) version >= 6 (which includes npm)
+* Install the LTS version of [NodeJS](https://nodejs.org/)
 
-* Install [Yarn](https://yarnpkg.com/) version >= 1.0.2, follow the instructions on the website or install it with npm:
```

Comment:
Come to think of it, I think it's important that you leave this comment ("follow the instructions on the website") because each platform has its own "right way" to install Yarn. Using `curl` should be the last alternative

Reply:
Fair point. If there's no outright best way to install yarn, we should merely point people to the website. Paging @erwinmombay and @choumx for their opinions.

Reply:
ok - LGTM on my end, once you get a consensus on this one. Don't forget to update both getting-started-*.md files :)

Reply:
agree with @danielrozenberg

Reply:
Fixed. We now just point users to the website.